scipy/stats/_multivariate.py

```diff
@@ -786,6 +786,39 @@ def entropy(self, mean=None, cov=1):
         dim, mean, cov_object = self._process_parameters(mean, cov)
         return 0.5 * (cov_object.rank * (_LOG_2PI + 1) + cov_object.log_pdet)
 
+    def fit(self, x):
```

Comment:
Ability to fix mean and covariance would be [added in a follow-up](https://github.com/scipy/scipy/issues/15676#issuecomment-1526459598).